applications/GeoMechanicsApplication/custom_elements/geo_steady_state_Pw_piping_element.h

```diff
@@ -0,0 +1,297 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Wijtze Pieter Kikstra
+//
+
+#pragma once
+
+#include "custom_utilities/dof_utilities.h"
+#include "custom_utilities/element_utilities.hpp"
+#include "custom_utilities/transport_equation_utilities.hpp"
+#include "geo_mechanics_application_variables.h"
+#include "includes/cfd_variables.h"
+#include "includes/element.h"
+#include "includes/serializer.h"
+
+namespace Kratos
+{
+
+template <unsigned int TDim, unsigned int TNumNodes>
+class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoSteadyStatePwPipingElement : public Element
+{
+public:
+    KRATOS_CLASS_INTRUSIVE_POINTER_DEFINITION(GeoSteadyStatePwPipingElement);
+
+    explicit GeoSteadyStatePwPipingElement(IndexType NewId = 0) : Element(NewId) {}
+
+    GeoSteadyStatePwPipingElement(IndexType NewId, GeometryType::Pointer pGeometry)
+        : Element(NewId, pGeometry)
+    {
+    }
+
+    GeoSteadyStatePwPipingElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+        : Element(NewId, pGeometry, pProperties)
+    {
+    }
+
+    Element::Pointer Create(IndexType NewId, const NodesArrayType& rThisNodes, PropertiesType::Pointer pProperties) const override
+    {
+        return make_intrusive<GeoSteadyStatePwPipingElement>(NewId, GetGeometry().Create(rThisNodes), pProperties);
```

Comment:
Perhaps we can express this overload in terms of the other:
```suggestion
        return this->Create(NewId, GetGeometry().Create(rThisNodes), pProperties);
```

Reply:
done